src/Storage/Storage/Blob/Cmdlet/RemoveAzureStorageContainer.cs

```diff
@@ -27,7 +27,6 @@ namespace Microsoft.WindowsAzure.Commands.Storage.Blob.Cmdlet
     /// remove specified azure container
     /// </summary>
     [Cmdlet("Remove", Azure.Commands.ResourceManager.Common.AzureRMConstants.AzurePrefix + "StorageContainer", SupportsShouldProcess = true),OutputType(typeof(Boolean))]
-    [Alias("Remove-" + Azure.Commands.ResourceManager.Common.AzureRMConstants.AzurePrefix + "DatalakeGen2FileSystem")]
```

Comment:
@blueww confirmed, not alias for previous sprint.

Reply:
Confirm: This alias only on preview module Az.Storage before.